src/test/java/tests/FilterEvalTests.java

```diff
@@ -124,6 +124,20 @@ public void in() {
         assertEquals(false, matches("in:something a is", issue));
     }
 
+    private void milestoneTest(String milestoneQualifier, TurboIssue issue, IModel model) {
```

Comment:
Could be helpful if tests and their helper methods like these have comments or more informative names saying what they are trying to test so it's easier to check for coverage and also to fix or refactor them in the future.